app/validators/file_upload_validator.rb

```diff
@@ -0,0 +1,57 @@
+class FileUploadValidator < ActiveModel::Validator
+  MIN_FILE_SIZE = 5 # KB
+  MAX_FILE_SIZE = 20 # MB
+
+  ALLOWED_CONTENT_TYPES = %w[
+    application/msword
+    application/pdf
+    application/rtf
+    application/vnd.ms-excel
+    application/vnd.oasis.opendocument.text
+    application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
+    application/vnd.openxmlformats-officedocument.wordprocessingml.document
+    image/bmp
+    image/gif
+    image/jpeg
+    image/png
+    image/tiff
+    image/x-bitmap
+    text/csv
```

Comment:
If we're going to allow these, we need to double check that's ok, and update the copy in the designs to say csv

Reply:
Civil does allow (if that was the intention, to match what they do)